unlock-app/package.json

```diff
@@ -17,6 +17,7 @@
     "color-scheme": "^1.0.1",
     "connected-react-router": "^6.1.0",
     "cross-env": "^5.2.0",
+    "ethereumjs-util": "^6.0.0",
```

Comment:
separate PRs for config, 1 to add ethereumjs-util, and 1 to remove uniqid after this one is merged

Reply:
Yup! Thanks!

Reply:
This feedback has now been addressed